web/src/main/java/com/epam/esm/controller/RoleController.java

```diff
@@ -0,0 +1,26 @@
+package com.epam.esm.controller;
+
+import com.epam.esm.entity.Role;
+import com.epam.esm.service.impl.RoleServiceImpl;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping(value = "/api/roles", produces = "application/json", consumes = "application/json")
+public class RoleController {
+    private final RoleServiceImpl roleService;
+
+    public RoleController(RoleServiceImpl roleService) {
+        this.roleService = roleService;
+    }
+
+    @PostMapping()
+    @PreAuthorize("hasRole('ADMIN')")
+    public ResponseEntity<Role> saveTag(@RequestBody Role role) {
```

Comment:
Seems to me, method name is wrong here. It should be saveRole.
My question is why do you need API to create new roles? I think it is useless.